src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -71,6 +71,7 @@
         "symfony/uid": "^5.4|^6.0|^7.0",
         "symfony/web-link": "^5.4|^6.0|^7.0",
         "phpdocumentor/reflection-docblock": "^3.0|^4.0|^5.0",
+        "dragonmantank/cron-expression": "^3.1",
```

Comment:
You need to move it to keep the list ordered.
Also, you must add it to the main `composer.json` file as well.

Reply:
I suspected that but then saw `phpdocumentor` after `symfony` :thinking: 

Fixed, and it is already added to main composer.json previously